src/client/components/RoomSelector.jsx

```diff
@@ -57,11 +52,15 @@ const RoomSelector = ({socket, joinRoom}) => {
     {!currentRoomId ?
       <form className={`create-room`} onSubmit={createRoom}>
         <input placeholder="Room name..."
+          autoFocus
+          maxLength="50"
           name="roomName"
           ref={roomNameInput}
           className="neon-border"
-          onInput={ev => setRoomName(ev.target.value)}
-          required />
+          onInput={ev => setRoomName(ev.target.value.trim())}
```

Comment:
I thought I'd be smart and retrieve the input value from the form directly in the onSubmit handler, so it doesn't reuse this value. You might want to change `createRoom` so that it uses the state `roomName` rather than getting it from the form data. At the moment, `roomName` is only used to show the button as disabled, so it will still happily ask the server to create a room with an untrimmed name (the server will now trim it so it's no big deal but it's inconsistent with how things are done for the `userName`)